tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,74 @@
+    <command detect_errors="aggressive">
+<![CDATA[
+        python '$__tool_directory__/main.py'
+
+            --imaging_model '$input_imaging_model'
+            --image_file '$input_image_file'
+	    --image_size '$input_image_input_size'
+
+]]>
+    </command>
+    <inputs>
+        <param name="input_imaging_model" type="data" format="zip" label="BioImage model" help="Please load a BioImage model from file uploader"/>
+	<param name="input_image_file" type="data" format="tif" label="Image to be analysed" help="Please provide an image"/>
+	<param name="input_image_input_size" type="text" label="Shape of the input image" help="Provide input shape as a tuple. See chosen model's RDF file to find correct input size. For example: Mitochondria EM Segmentation BoundaryModel, the input size is 256 x 256 x 32 x 1. Enter the size as (256,256,32,1)."/>
+    </inputs>
+    <outputs>
+	<data format="tif" name="output_predicted_image" from_work_dir="output_predicted_image.tif" label="Predicted image"></data>
+	<data format="npy" name="output_predicted_image_matrix" from_work_dir="output_predicted_image_matrix.npy" label="Predicted image matrix"></data>
+    </outputs>
+    <tests>
+	<test>
+	    <param name="input_imaging_model" value="input_imaging_model.zip" location="https://zenodo.org/api/records/6647674/files/weights-torchscript.pt/content"/>
+	    <param name="input_image_file" value="input_image_file.tif" location="https://zenodo.org/api/records/6647674/files/sample_input_0.tif/content"/>
+	    <param name="input_image_input_size" value="256,256,1,1"/>
+	    <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="sim_size" delta="100" />
+	    <output name="output_predicted_image_matrix" file="output_nucleisegboundarymodel_matrix.npy" compare="sim_size" delta="100" />
+        </test>
+    </tests>
+    <help>
+        <![CDATA[
+**What it does**
+
+The tool takes a BioImage model and an image to be analysed. Model makes a prediction and the predicted image becomes available as a TIF file.
+
+**Description**
+The tool takes a BioImage model and an image to be analysed. Model makes a prediction and the predicted image becomes available as a TIF file.
+
+**Input files**
+	- BioImage model: Add one of the model from Galaxy file uploader by choosing a "femote" file at "ML Models/bioimaging-models"
+	- Image to be analysed: Pass an image as TIF file
+	- Provide the necessary input shape for the model. This information can be found in the RDF file of each model (RDF file > config > test_information > inputs > size)
+
+**Output file**
+        - Predicted image: Predicted image using the Bioimage model
+        - Predicted image matrix: Predicted image matrix in original dimensions
+        ]]>
+    </help>
+    <citations>
+        <citation type="bibtex">
+            @ARTICLE{anuprulez_galaxytools,
+                Author = {Anup Kumar and Beatriz Serrano-Solano and Björn Grüning},
```

Comment:
@kostrykin you should be there too :)

Reply:
This format of citation has been replaced by two DOIs based ones.

Reply:
Thanks, but I think this is what the [creator fields](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator) are actually meant for.

Reply:
I have added some information inside `<creator>` tag: https://github.com/bgruening/galaxytools/blob/9961ea5c779320b832154afca06432d1eea6a9d9/tools/bioimaging/bioimage_inference.xml#L7

But, I am not sure about a few fields